gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+@Component({
+  selector: "app-expandable",
+  templateUrl: "./expandable.component.html",
+  styleUrls: ["./expandable.component.scss"],
+})
+export class ExpandableComponent implements OnInit {
+
+  @Input() signalMessage;
+  itemExpanded = false;
+
+  constructor() { }
+
+  ngOnInit() {}
+
+  async toggleFullMessage() {
```

Comment:
Why is this async?

Reply:
Shouldn't be, leftover from an earlier version. Good catch.